### PR TITLE
[Exporter.Geneva] Add support for exporting metrics to different combinations of account and namespace

### PR DESCRIPTION
**Pull request by utpilla**
## Changes
- Add support for exporting metrics to different combinations of account and namespace

Please provide a brief description of the changes here.

For significant contributions please make sure you have completed the following items:

* [x] Appropriate `CHANGELOG.md` updated for non-trivial changes

### CODE REVIEW

File: src/OpenTelemetry.Exporter.Geneva/Metrics/GenevaMetricExporter.cs [hunk 1]
```diff
@@ -119,6 +123,9 @@ public GenevaMetricExporter(GenevaMetricExporterOptions options)
 
     public override ExportResult Export(in Batch<Metric> batch)
     {
+        string monitoringAccount = this.defaultMonitoringAccount;
```
Comment (cijothomas) [inline src/OpenTelemetry.Exporter.Geneva/Metrics/GenevaMetricExporter.cs:126]: as an enhancement, consider making this optional. i.e users can avoid specifying account, ns in connectionstring.

File: src/OpenTelemetry.Exporter.Geneva/Metrics/GenevaMetricExporter.cs [hunk 2]
```diff
@@ -885,6 +929,31 @@ private static void SerializeHistogramBucketWithTLV(in HistogramBucket bucket, b
         MetricSerializer.SerializeUInt32(buffer, ref bufferIndex, Convert.ToUInt32(bucket.BucketCount));
     }
 
+    [MethodImpl(MethodImplOptions.AggressiveInlining)]
+    private void GetCustomAccountAndNamespaceIfAny(in ReadOnlyTagCollection tags, out string monitoringAccount, out string metricNamespace)
+    {
+        monitoringAccount = this.defaultMonitoringAccount;
+        metricNamespace = this.defaultMetricNamespace;
+
+        foreach (var tag in tags)
```
Comment (cijothomas) [inline src/OpenTelemetry.Exporter.Geneva/Metrics/GenevaMetricExporter.cs:938]: we are iterating several times over tags. Check if we can avoid it.

File: src/OpenTelemetry.Exporter.Geneva/CHANGELOG.md [hunk 3]
```diff
@@ -2,10 +2,17 @@
...
 ## Unreleased
 
+* Fix a bug where metrics without exemplars were not getting exported.
+  ([#1099](https://github.com/open-telemetry/opentelemetry-dotnet-contrib/pull/1099))
+
 * Relaxed table name mapping validation rules to restore the previous behavior
   from version 1.3.0.
   ([#1109](https://github.com/open-telemetry/opentelemetry-dotnet-contrib/issues/1109))
 
+* Add support for exporting metrics to more than a single account/namespace
```
Comment (cijothomas) [inline src/OpenTelemetry.Exporter.Geneva/CHANGELOG.md:12]: please mention here the name of dimension keys which should contain account and namespace.

**Review by cijothomas — APPROVED** [review]
LGTM

File: src/OpenTelemetry.Exporter.Geneva/Metrics/GenevaMetricExporter.cs (hunk 1 above)
Comment (utpilla) [reply src/OpenTelemetry.Exporter.Geneva/Metrics/GenevaMetricExporter.cs:126]: I'll do this in a follow up PR.

File: src/OpenTelemetry.Exporter.Geneva/Metrics/GenevaMetricExporter.cs (hunk 2 above)
Comment (utpilla) [reply src/OpenTelemetry.Exporter.Geneva/Metrics/GenevaMetricExporter.cs:919]: Fixed in https://github.com/open-telemetry/opentelemetry-dotnet-contrib/pull/1111/commits/85d03d139815a64ee8278607bc91be091d822e0f.

**Review by cijothomas — APPROVED** [review]

**Pull request merged**